pkgs/development/python-modules/configupdater/default.nix

```diff
@@ -0,0 +1,37 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pytestCheckHook
+, setuptools-scm
+}:
+
+buildPythonPackage rec {
+  pname = "configupdater";
+  version = "3.1.1";
+
+  src = fetchPypi {
+    inherit version;
+    pname = "ConfigUpdater";
+    sha256 = "sha256-RvDHTXPvpyN3Z2S0PJc59oBSSV3T1zQxnB0OtYUR8Vs=";
+  };
+
+  postPatch = ''
+    substituteInPlace setup.cfg \
+      --replace '--cov' '#--cov'
```

Comment:
```suggestion
      --replace "--cov configupdater --cov-report term-missing" ""
```

Reply:
Yes I guess in a future version they could append something after `--cov` that's useful